httpserver/https_client.cpp

```diff
@@ -28,23 +31,49 @@ void make_https_request(boost::asio::io_context& ioc,
         return;
     }
 
+    const std::string sn_pubkey_b32z =
+        std::regex_replace(sn_address, std::regex("\\.snode"), "");
```

Comment:
We now have pubkey (without ".snode") in `sn_record_t`. You can get it by calling `sn_record_t::pub_key`.